### PR TITLE
Ask for confirmation if receiver address is not the user address

### PR DESCRIPTION
**Pull request by pmprete**
We previously didn't allowed claims when the receiver address was not the user, this was donde to avoid sending it to a wrong address
A user sent funds directly to their exchange, we do want to allow this kind of behaviour, so we added a modal asking for confirmations when sending to another address

https://user-images.githubusercontent.com/3655076/128203161-7791cb67-e57a-4113-8afa-f9600eadef6d.mov

### CODE REVIEW

File: src/components/transactions/TransactionRow.vue [hunk 1]
```diff
@@ -393,6 +402,27 @@ export default {
           console.error(err)
         })
     },
+    async claimClick() {
+      const data = this
```
Comment (marcciosilva) [inline src/components/transactions/TransactionRow.vue:406]: maybe I'd extract `data.sharedState` to a `let sharedState = data.sharedState` since we're using it multiple times (and writing to it) below

File: src/constants/tokens.js [hunk 2]
```diff
@@ -180,7 +180,7 @@ const UBI_TOKEN = {
   typeId: 6,
   icon: 'https://raw.githubusercontent.com/rsksmart/rsk-contract-metadata/master/images/ubi.png',
   30: { symbol: 'rUBI', address: '0x70566d8541beabe984c8babf8a816ed908514ba8', decimals: 18 },
-  31: { symbol: 'rUBI', address: '0x1c85316116131133cf6fdb0cde836b23035ea775', decimals: 18 },
```
Comment (marcciosilva) [inline src/constants/tokens.js:183]: so we had rUBI duplicated? I guess the frontend table shows unique symbol values since I don't see it twice in the frontend

File: src/components/transactions/TransactionRow.vue [hunk 3]
```diff
@@ -62,6 +62,31 @@
         <p>{{ connectionProblem }}</p>
       </template>
     </Modal>
+    <Modal v-if="showMissmatchAddressModal" @close="showMissmatchAddressModal = false">
+      <template #title>
+        Receiver address is not the wallet
```
[the remark this answers is not in the document]
Comment (pmprete) [reply src/components/transactions/TransactionRow.vue:67]: actually, the connected wallet only has one address we can see, he has to change accounts in order for us to see the other addresses. 
Maybe rewording it to  `Receiver address is not the connected account` ?

File: src/components/transactions/TransactionRow.vue [hunk 4]
```diff
@@ -62,6 +62,31 @@
...
+      </template>
+      <template #body>
+        <p>
+          The receiver address {{ transaction.receiverAddress }} is not your currently connected
```
[the remark this answers is not in the document]
Comment (pmprete) [reply src/components/transactions/TransactionRow.vue:71]: Rewording it to currently connected account

File: src/components/transactions/TransactionRow.vue [hunk 5]
```diff
@@ -62,6 +62,31 @@
...
+          Are you sure you want to claim the funds to {{ transaction.receiverAddress }} anyway?
+        </p>
+      </template>
+      <template #footer>
+        <button class="btn btn-primary modal-default-button" @click="claim()">
+          Yes
+        </button>
+        <button
+          class="btn btn-danger modal-default-button"
+          @click="showMissmatchAddressModal = false"
```
[the remark this answers is not in the document]
Comment (pmprete) [reply src/components/transactions/TransactionRow.vue:84]: Deam typos, fixing it

File: src/components/transactions/TransactionRow.vue [hunk 6]
```diff
@@ -309,26 +335,9 @@ export default {
         }
       }
     },
-    async claimClick() {
+    async claim() {
       const data = this
-      if (!data.sharedState.isConnected) {
-        data.connectionProblem = `Wallet not connected. You need to connect your wallet to ${data.toNetwork.name}`
```
[the remark this answers is not in the document]
Comment (pmprete) [reply src/components/transactions/TransactionRow.vue:315]: This is kept, just chnaged the line where it's happening https://github.com/rsksmart/tokenbridge-ui/pull/20/files#diff-5012711c100f01321c707e9ee47990ae3c92cc2215e3f6a007ae6cb68381536dR408

File: src/components/transactions/TransactionRow.vue (hunk 1 above)
Comment (pmprete) [reply src/components/transactions/TransactionRow.vue:406]: ok

File: src/constants/tokens.js (hunk 2 above)
Comment (pmprete) [reply src/constants/tokens.js:183]: They are on diferent networks so it's not duplicated, the error is the displaying Symbol for ir on rskTestnet, it should have been rKovUBI instead of rUBI

File: src/components/transactions/TransactionRow.vue (hunk 1 above)
Comment (pmprete) [reply src/components/transactions/TransactionRow.vue:402]: done

File: src/components/transactions/TransactionRow.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (marcciosilva) [reply src/components/transactions/TransactionRow.vue]: sounds good!

File: src/components/transactions/TransactionRow.vue (hunk 6 above)
[the remark this answers is not in the document]
Comment (marcciosilva) [reply src/components/transactions/TransactionRow.vue:315]: ohh missed it, thanks!

**Review by marcciosilva — APPROVED** [review]
lgtm!

**Review by jjhcuadrosIOV — APPROVED** [review]
Good job
